DirectEmployers/MyJobs
# [PD-3286] Fixing Broken Breadcrumb Navigation

## What's this PR do?
PR fixes the breadcrumb navigation that's broken inside of Analytics

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start inside any of the analytics applications

## How should this be manually tested?
Pull down repository. Go inside any of the analytics apps and filter down. Click on the breadcrumbs and make sure they are going to the right active tab for that breadcrumb

## Any background context you want to provide?
